src/Raven.Server/Documents/Handlers/Debugging/MemoryDebugHandler.cs

```diff
@@ -55,17 +55,27 @@ static DynamicJsonValue ToJson(GCMemoryInfo info)
                     [nameof(info.FragmentedBytes)] = info.FragmentedBytes,
                     ["FragmentedHumane"] = Size.Humane(info.FragmentedBytes),
                     [nameof(info.Generation)] = info.Generation,
-                    [nameof(info.GenerationInfo)] = new DynamicJsonArray(info.GenerationInfo.ToArray().Select(x => new DynamicJsonValue
-                    {
-                        [nameof(x.FragmentationAfterBytes)] = x.FragmentationAfterBytes,
-                        ["FragmentationAfterHumane"] = Size.Humane(x.FragmentationAfterBytes),
-                        [nameof(x.FragmentationBeforeBytes)] = x.FragmentationBeforeBytes,
-                        ["FragmentationBeforeHumane"] = Size.Humane(x.FragmentationBeforeBytes),
-                        [nameof(x.SizeAfterBytes)] = x.SizeAfterBytes,
-                        ["SizeAfterHumane"] = Size.Humane(x.SizeAfterBytes),
-                        [nameof(x.SizeBeforeBytes)] = x.SizeBeforeBytes,
-                        ["SizeBeforeHumane"] = Size.Humane(x.SizeBeforeBytes)
-                    })),
+                    [nameof(info.GenerationInfo)] = new DynamicJsonArray(
+                        info.GenerationInfo.ToArray().Select((x, index) => new DynamicJsonValue
+                        {
+                            ["GenerationName"] = index switch
+                                {
+                                    0 => "Heap Generation 0",
+                                    1 => "Heap Generation 1",
+                                    2 => "Heap Generation 2",
+                                    3 => "Large Object Heap",
+                                    4 => "Finalization Queue",
```

Comment:
isn't that POH? I might be wrong.

Reply:
I couldn't find any official documentation on this. 
But if we go by the information mentioned [here](https://stackoverflow.com/questions/66861856/what-do-the-gcgenerationinfo-instances-in-gcmemoryinfo-generationinfo-represent) and match the data from the endpoint with the data displayed in dotMemory, then you're right - it's the Pinned Object Heap.
Fixed.